.readthedocs.yaml

```diff
@@ -0,0 +1,42 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+# Required
+version: 2
+
+build:
```

Comment:
We need ReadTheDocs to install the same packages that we do in the first stage of CI. I suspect some of these can be removed.